docs/streams/upgrade-guide.html

```diff
@@ -52,6 +52,15 @@ <h1>Upgrade Guide and API Changes</h1>
         <li> restart all new ({{fullDotVersion}}) application instances </li>
     </ul>
 
+    <p>
+        Since 3.0.0 release, Kafka Streams uses a newer RocksDB version which bumped its footer version persisted on files.
```

Comment:
A minor comment: Wouldn't it be simpler to just state that the on-disk format changed? Mentioning the footer version puts too much details in this guide, IMO.

Reply:
Ack.